src/Talis/Persona/Client/Base.php

```diff
@@ -1,7 +1,10 @@
 <?php
 namespace Talis\Persona\Client;
+require __DIR__ . '/../../../../vendor/autoload.php';
```

Comment:
You'll still obviously need to run composer and all that jazz in your project directory for development, but that gets you out of the business of worrying about what the ultimate deployment does.

We had a terrible situation with Tripod where it required its own composer install within its place in the vendor directory because it had all of these sorts of references to its dependencies (namely ARC2).